apps/core/templates/robots.txt

```diff
@@ -0,0 +1,6 @@
+User-Agent: *
+Disallow: /admin/
+Disallow: /search/
```

Comment:
I’m not clear on the value of disallowing this. If we retain a standalone search page, we’d probably want it crawled. If we don’t retain the page, we don’t need it listed here.